main.go

```diff
@@ -140,6 +142,17 @@ var (
 	membersCache []User
 )
 
+// blacklistedMemebers returns a list of login names of members which should
+// not be included in pull requests.
+func blacklistedMembers() map[string]bool {
+	ret := map[string]bool{}
```

Comment:
check out 'exploiting zero values' in -> https://blog.golang.org/go-maps-in-action